cdisc_rules_engine/operations/variable_is_null.py

```diff
@@ -0,0 +1,9 @@
+from cdisc_rules_engine.operations.base_operation import BaseOperation
+
+
+class VariableIsNull(BaseOperation):
+    def _execute_operation(self):
+        target_variable = self.params.target.replace("--", self.params.domain, 1)
```

Comment:
We probably want to return a default value if the target_variable is not in the dataset.